sample/src/main/java/co/nimblehq/sample/SampleApp.kt

```diff
@@ -0,0 +1,18 @@
+package co.nimblehq.sample
+
+import android.app.Application
+import co.nimblehq.survey.sdk.SurveyApi
+
+class SampleApp : Application() {
+
+    override fun onCreate() {
+        super.onCreate()
+        SurveyApi.instance
+            .withMode(BuildConfig.DEBUG)
+            .withBaseUrl(BuildConfig.BASE_URL)
+            .withCredentials(
+                clientID = BuildConfig.CLIENT_ID,
+                clientSecret = BuildConfig.CLIENT_SECRET
+            )
```

Comment:
In case the credentials have a lot of properties so we need to put them all in here. That's why I recommend creating a new object for this.

Reply:
@hoangnguyen92dn Okay, I agree with this.